### DISCUSSION TITLE
Are "GetDroppedFiles" and "ClearDroppedFiles" functions abandoned in raylib 4.0

### DISCUSSION THREAD

**Discussion by wlxmhls** [raysan5/raylib#2654] [Q&A] [answered]
Hello, I am trying the example code "models_loading.c". It uses the file dropping function. But it seems the GetDroppedFiles and ClearDroppedFiles functions are not available in raylib 4.0.
If the functions are abandoned, is there alternative functions? Thanks!

**Comment by raysan5** [comment, marked as answer]
@wlxmhls Those functions have been replaced by `LoadDroppedFiles()` and `UnloadDroppedFiles()`, now they are more consistent with other functions in the library. Examples have been properly updated, you can check them on GitHub.

**Reply by wlxmhls** [reply]
Thanks!

**Comment by wlxmhls** [comment]
external folder missing 'physac.h', which is physics engine header file, needed by examples of physics.

Hope the raylib gets better and better.
